app/styles/ui/history/_commit-summary.scss

```
@@ -104,7 +104,7 @@
   &-description-scroll-view {
     overflow: hidden;
     // The extra pixel makes the space align up with the commit list.
-    max-height: 69px;
+    max-height: 70px;
```

I've been investigating and I think this set of CSS properties would make the solution work in case we change the layout again:
```suggestion
    display: -webkit-box;
    -webkit-box-orient: vertical;
    -webkit-line-clamp: 3;
```
No more need for pixel perfect updates!

---

I just tried out these changes in DevTools (because I don't have Electron set up) and it doesn't seem to actually expand when I click the button. Let me know if I'm doing it wrong.

https://user-images.githubusercontent.com/75134774/174136876-6ea87d4d-3265-442e-99ba-17f36721cec3.mp4

---

Weird, it worked when I tested it 🤔 

In any case, l played a bit more with it, and you need to make this other change to the `expanded` class:
```diff
diff --git a/app/styles/ui/history/_commit-summary.scss b/app/styles/ui/history/_commit-summary.scss
index 704ba0762..4ad369c05 100644
--- a/app/styles/ui/history/_commit-summary.scss
+++ b/app/styles/ui/history/_commit-summary.scss
@@ -20,20 +20,21 @@
     svg {
       vertical-align: text-top;
       margin-right: var(--spacing-half);
     }
   }

   &.expanded {
     .commit-summary-description-scroll-view {
       max-height: 400px;
       overflow: auto;
+      display: revert;

       &:before {
         content: none;
       }
     }
   }

   &.hide-description-border {
     .commit-summary-description-container {
       border-bottom: none;
```

This will revert the `display: -webkit-box;` from the other set of changes and will effectively expand the text:

https://user-images.githubusercontent.com/1083228/174261558-0930f87e-7b6d-474a-9335-64ad107aa038.mov

---

I'm confused about what lines you actually changed, because your diff only states `display: revert` but the rest of the attributes are completely different on my system (i.e. `max-height: 400px` on `#commit-summary.expanded` which is `none` here).

I did my best to make my CSS as close as possible to yours and it still doesn't work.

Tomorrow I'll setup a dev environment and try editing the source .scss files instead of using DevTools to see if it will work.

---

Sorry, I just tried again and it worked. Please disregard the comment above.

I'll be doing more tests tomorrow and committing the new changes.